<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Particula</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="id_label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="id_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="origen_x_label">
         <property name="text">
          <string>Origen X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="origen_x_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="origen_y_label">
         <property name="text">
          <string>Origen Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="origen_y_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="destino_x_label">
         <property name="text">
          <string>Destino X:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="destino_x_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="destino_y_label">
         <property name="text">
          <string>Destino Y:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="destino_y_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="velocidad_label">
         <property name="text">
          <string>Velocidad:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="velocidad_spinBox">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="red_label">
         <property name="text">
          <string>Red:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="red_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="green_label">
         <property name="text">
          <string>Green:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="green_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="blue_label">
         <property name="text">
          <string>Blue:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="blue_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="agregar_inicio_particula_pushButton">
         <property name="text">
          <string>Agregar al inicio</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QPushButton" name="agregar_final_particula_pushButton">
         <property name="text">
          <string>Agregar al final</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Salida</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="salida"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mostrar_particula_pushButton">
            <property name="text">
             <string>Mostrar particulas</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="mostrar_grafo_pushButton">
            <property name="text">
             <string>Mostrar grafo</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Visualizar">
       <attribute name="title">
        <string>Visualizador</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Tabla">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Identificador de p√°rticula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuOrdenar">
    <property name="title">
     <string>Ordenar</string>
    </property>
    <addaction name="actionID_ascendente"/>
    <addaction name="actionDistancia_descendente"/>
    <addaction name="actionVelocidad_ascendente"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionPuntos_cercanos"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionPuntos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuOrdenar"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionID_ascendente">
   <property name="text">
    <string>ID (ascendente)</string>
   </property>
  </action>
  <action name="actionDistancia_descendente">
   <property name="text">
    <string>Distancia (descendente)</string>
   </property>
  </action>
  <action name="actionVelocidad_ascendente">
   <property name="text">
    <string>Velocidad (ascendente)</string>
   </property>
  </action>
  <action name="actionPuntos_cercanos">
   <property name="text">
    <string>Puntos Cercanos</string>
   </property>
  </action>
  <action name="actionPuntos">
   <property name="text">
    <string>Puntos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
